GitUI/Translation/English.xlf

```diff
@@ -111,10 +111,6 @@ This action will be performed without warning while checking out branch.</source
   </file>
   <file datatype="plaintext" original="AppearanceSettingsPage" source-language="en">
     <body>
-      <trans-unit id="$this.Text">
-        <source>Appearance</source>
-        <target />
-      </trans-unit>
```

Comment:
This must not change. Please rebase on to the latest master

Reply:
Revert this change to make the build pass.

Reply:
Fixed.